webpack.config.js

```diff
@@ -0,0 +1,65 @@
+const HtmlPlugin = require('html-webpack-plugin');
+const CleanPlugin = require('clean-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    filename: 'bundle.[hash].js'
+  },
+  devServer: {
+    port: 7890
+  },
+  plugins: [
+    new HtmlPlugin({ template: './src/index.html' }),
+    new CleanPlugin('./dist')
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-env', '@babel/preset-react'],
```

Comment:
```suggestion
            presets: ['@babel/preset-env', '@babel/preset-react'],
            plugins: ['@babel/plugin-proposal-class-properties'],
```
Add the line above to the babel-loader options object.